describe_groups_response.go

```diff
@@ -0,0 +1,162 @@
+package sarama
+
+type DescribeGroupsResponse struct {
+	Groups []*GroupDescription
```

Comment:
kinda/almost/sorta makes sense for this to be a map keyed by group ID, but not sure if it's worth it at this point